src/test/scala-2/chiselTests/experimental/hierarchy/SeparateElaborationSpec.scala

```diff
@@ -503,4 +503,45 @@ class SeparateElaborationSpec extends AnyFunSpec with Matchers with Utils with T
       "Expected distinct overrideDef names but found duplicates for: Inst1_Prefix_AddOnePrameterized_Inst1_Suffix"
     )
   }
+
+  class Bar extends Module {
+    val a = IO(Input(Bool()))
+    layer.block(layers.Verification.Assert) {
+      chisel3.assert(a === true.B)
+    }
+  }
+
+  describe("(5): Known layers") {
+
+    it("(5.a): should be added to imported definitions") {
+
+      val testDir = implicitly[HasTestingDirectory].getDirectory.toString
+
+      val annotations = (new ChiselStage).execute(
+        Array("--target-dir", s"$testDir/Bar", "--target", "chirrtl"),
+        Seq(ChiselGeneratorAnnotation(() => new Bar))
+      )
```

Comment:
Is there any reason to use `class ChiselStage` instead of just `ChiselStage.emitCHIRRTLFile`? (it does take `args`).

Reply:
I thought we didn't have a method that returns an `AnnotationSeq`, but `emitCHIRRTLFile` looks like it does this. 👍